dll/win32/shell32/dialogs/drvdefext.cpp

```diff
@@ -538,10 +538,13 @@ CDrvDefExt::GeneralPageProc(
                                  &cbBuf) == ERROR_SUCCESS)
                 {
                     WCHAR wszCmd[MAX_PATH];
+                    WCHAR TempArr[MAX_PATH] = { 0 };
 
-                    StringCbPrintfW(wszCmd, sizeof(wszCmd), wszBuf, pDrvDefExt->m_wszDrive[0]);
+                    StringCbPrintfW(TempArr, sizeof(TempArr), L"/c %s", wszBuf);
+                    
+                    StringCbPrintfW(wszCmd, sizeof(wszCmd), TempArr, pDrvDefExt->m_wszDrive[0]);
 
-                    if (ShellExecuteW(hwndDlg, NULL, wszCmd, NULL, NULL, SW_SHOW) <= (HINSTANCE)32)
+                    if (ShellExecuteW(hwndDlg, L"open", L"cmd", wszCmd, NULL, SW_HIDE) <= (HINSTANCE)32)
```

Comment:
Why open a command prompt here?

Reply:
@learn-more Unless if there is other way to do so, ShellExecuteW will not work with path containing `%SystemRoot%` gathered from a registry value. And since I wanted to keep the compatibility I had to go with this way.

Reply:
yeah this is a huge hack.
Use something like ExpandEnvironmentStrings or so...